test/bdd/go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/hyperledger/aries-framework-go v0.0.0
 	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
 	github.com/morikuni/aec v1.0.0 // indirect
+	github.com/mr-tron/base58 v1.1.3
```

Comment:
We should probably replace this library with `github.com/btcsuite/btcutil/base58`

Reply:
I am using `btcsuite/btcutil/base58`. I forgot to clean `go.mod` file